src/instana/version.py

```diff
@@ -3,4 +3,4 @@
 
 # Module version file.  Used by setup.py and snapshot reporting.
 
-VERSION = "3.5.0"
+VERSION = "3.5.0.post1"
```

Comment:
Shouldn't be `3.5.1-fedramp` ?

```suggestion
VERSION = "3.5.1-fedramp"
```

Reply:
Hi @pvital , related slack conversation: [link](https://instana.slack.com/archives/C06A6H9EKHS/p1754473681812429?thread_ts=1754473476.101859&cid=C06A6H9EKHS)